pages/main.tsx

```diff
@@ -2,10 +2,11 @@
 // SPDX-License-Identifier: Apache-2.0
 import { StrictMode } from "react";
 import ReactDOM from "react-dom/client";
-import "@cloudscape-design/global-styles/index.css";
 
 import App from "./app";
 
+import "@cloudscape-design/global-styles/index.css";
```

Comment:
According to the eslint grouping shouldn't this come before the relative import at line `6` 🤔 
I think this is an overlap between the `^@cloudscape` and `*.css` rule.

Reply:
file extension takes precedence in this case, all styles should come last